dart_conf_flutter/lib/src/schedule.dart

```diff
@@ -309,6 +309,8 @@ class SessionCardWidget extends StatelessWidget {
 class SessionPage extends StatelessWidget {
   static final TextStyle titleStyle =
       const TextStyle(fontSize: 20.0, fontWeight: FontWeight.w600);
+  static final TextStyle dateTimeStyle =
```

Comment:
Not necessarily for this PR, but we'll probably want to share and reference the style constants from `SessionCardWidget`.

Reply:
We can leave it out for now, and refactor later.